.github/workflows/docker-clean.yaml

```diff
@@ -0,0 +1,48 @@
+on:
+    pull_request:
+      types:
+        - closed
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+    list-images:
```

Comment:
I don't know how to fix this, but this will leave some stragglers if the image tags were updated during the PR process (which will be most of the time).

I think we may want to add a manual workflow that just deletes any tag starting with `branch-` that we run occasionally. That will be the easiest solution at any rate.

Reply:
OK, I'm now using the GitHub API to get the IDs of any package with the `branch-<id>` prefix and any images without a tag. All of those are passed through for deletion.

Reply:
Looks great!